client/my-sites/plans/jetpack-plans/product-lightbox/included-product-list.tsx

```diff
@@ -0,0 +1,63 @@
+import { TranslateResult, useTranslate } from 'i18n-calypso';
+import getProductIcon from '../product-store/utils/get-product-icon';
+import slugToSelectorProduct from '../slug-to-selector-product';
+
+type IncludedProductsProps = {
+	products: ReadonlyArray< string >;
+	descriptionMap: Record< string, TranslateResult >;
+};
+
+type IncludedProductListItemProps = {
+	productSlug: string;
+	descriptionMap: Record< string, TranslateResult >;
+};
+
+const IncludedProductListItem: React.FC< IncludedProductListItemProps > = ( {
+	productSlug,
+	descriptionMap,
+} ) => {
+	const product = slugToSelectorProduct( productSlug );
+	return (
+		<div className="product-lightbox__included-product-list-item">
+			<div className="product-lightbox__included-product-list-item-icon">
+				<img alt="" src={ getProductIcon( { productSlug, light: true } ) } />
+			</div>
+
+			<div className="product-lightbox__included-product-list-item-content">
+				<h1 className="product-lightbox__included-product-list-item-title">
+					{ product?.shortName }
+				</h1>
+
+				<p className="product-lightbox__included-product-list-item-description">
+					{ descriptionMap[ productSlug ] }
+				</p>
+			</div>
+		</div>
+	);
+};
+
+const IncludedProductList: React.FC< IncludedProductsProps > = ( { products, descriptionMap } ) => {
+	const translate = useTranslate();
+
+	return (
+		<div className="product-lightbox__included-product-list">
+			<h3 className="product-lightbox__included-product-list-label">
+				{ translate( 'Products included:' ) }
+			</h3>
+
+			<ul>
+				{ products.map( ( productSlug ) => (
+					<li>
+						<IncludedProductListItem
+							productSlug={ productSlug }
+							descriptionMap={ descriptionMap }
+							key={ productSlug }
+						/>
+					</li>
```

Comment:
Here the `key` prop must be passed to `<li>` instead. Otherwise, you will see this warning there:

<img width="992" alt="Screenshot 2022-12-12 at 5 22 48 PM" src="https://user-images.githubusercontent.com/18226415/207038551-eeae19a1-ed9a-4943-a9d7-8319118a1b9f.png">